src/System.Data.SqlClient/src/System/Data/Sql/SqlSer.cs

```diff
@@ -244,5 +238,10 @@ public override void Write(byte[] buffer, int offset, int count)
         {
             _size += count;
         }
+
+        public override void Write(ReadOnlySpan<byte> buffer)
```

Comment:
Is this overload being added for the of completeness ? 
Since this class is internal and this overload is not being used, there is no point of adding this. 

Let me know if the intentions were not the above and if I am missing something.

Reply:
I explained that change here: https://github.com/dotnet/corefx/pull/36838#discussion_r275115142